addon/mixins/axis-titles.js

```diff
@@ -11,10 +11,31 @@ import Ember from 'ember';
  */
 const AxisTitlesMixin = Ember.Mixin.create({
   /**
-   * Toggle axis titles on/off
+   * Toggle axis X title on/off
    * @type {Boolean}
    */
-  hasAxisTitles: false,
+  hasXAxisTitle: false,
+
+  /**
+   * Toggle axis Y title on/off
+   * @type {Boolean}
+   */
+  hasYAxisTitle: false,
+
+  /**
+   * A deprecated property to support backward compatability.
+   * @deprecated
+   */
+  hasAxisTitles: Ember.computed('hasXAxisTitle', 'hasYAxisTitle',
```

Comment:
It would be nice if we could somehow use a deprecating alias (http://emberjs.com/api/classes/Ember.computed.html#method_deprecatingAlias), that then calls this function under the hood.